lesson_5/homeTask/index.html

```diff
@@ -1,65 +1,62 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>lesson_5 - HomeTask</title>
+    <title>lesson_5 - HomeTask version-1.7</title>
+    <link
+      rel="stylesheet"
+      href="C:/Robot_Dreams/lesson_7/main.css"
```

Comment:
no no no, it's a big mistake
cope the css file to your lesson_5 folder
and then it's enough just `main.css` in href